### PR TITLE
Adding button "Learn Kubernetes Basics"

### PR DESCRIPTION
**Pull request by truongnh1992**
Add missing button "Learn Kubernetes Basics". Resolve issue #23281

Signed-off-by: Nguyen Hai Truong <nguyenhaitruonghp@gmail.com>

### CODE REVIEW

**Review by sftim — COMMENTED** [review]
I can see that this page is missing, but I don't speak Vietnamese and so can't review the PR.

**Comment by truongnh1992** [comment]
> I can see that this page is missing, but I don't speak Vietnamese and so can't review the PR.

I just notified you that the missing button has been added. Thanks!
@huynguyennovem Please take your time to review this PR.

File: content/vi/docs/tutorials/kubernetes-basics/_index.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+        <p>Những hướng dẫn này dùng Katacoda để chạy một terminal ảo trên trình duyệt web của bạn chạy Minikube. Không cần phải cài đặt và cấu hình bất kỳ phần mềm nào; mỗi hướng dẫn tương tác chạy trực tiếp từ trình duyệt web của bạn.</p>
+      </div>
+    </div>
+
+    <br>
+
+    <div class="row">
+      <div class="col-md-9">
+        <h2>Kubernetes có thế làm những gì?</h2>
+        <p>Với các dịch vụ web hiện đại, người dùng mong muốn các ựng dụng luôn sẵn sàng hoạt động 24/7 và các lập trình viên muốn triển khai các phiên bản của ứng dụng đó nhiều lần trong ngày. Việc đóng gói ứng dụng vào container giúp giải quyết mục tiêu này, cho phép các ứng dụng được phát hành và cập nhật một cách dễ dàng, nhanh chóng mà không có downtime. Kubernetes giúp bạn đảm bảo các ứng dụng container chạy ở bất kì đâu và bất kì lúc nào bạn muốn, đồng thời giúp chúng tìm thấy các tài nguyên và công cụ cần thiết để chạy. Kubernetes là nền tảng mã nguồn mở, chạy được trong môi trường production, được thiết kế và phát triển bởi Google, kết hợp với những ý tưởng tốt nhất từ cộng đồng.</p>
```
Comment (huynguyennovem) [inline content/vi/docs/tutorials/kubernetes-basics/_index.html:43]: Please change this: "ựng" -> "ứng"

Comment (truongnh1992) [reply content/vi/docs/tutorials/kubernetes-basics/_index.html:43]: Thank you. Just fixed.

**Comment by Sea-n** [comment]
Hey @truongnh1992,

Seems you have open the branch in the wrong repo.

Could you delete this `add-button` branch from k/website? Thanks.

**Pull request merged**